src/code_1AF230.c

```diff
@@ -1,3 +1,21 @@
 #include "common.h"
 
-INCLUDE_ASM(s32, "code_1AF230", LoadStarPowerScript);
+typedef struct StarPower { void* dmaStart; void* dmaEnd; void* dmaDest; Bytecode* init; } StarPower;
+
+extern StarPower D_8029C7D0[13];
+
+typedef struct WhateverThisIs { s8 starpoints; s8 a; s8 b; s8 c; s32 w; s32 x; s32 y; s32 z; } WhateverThisIs;
```

Comment:
Name this "StarPowerDesc" or similar. The first field should be `starPoints` also